pkg/api/mlflow/service/artifact/validators_test.go

```diff
@@ -139,3 +139,156 @@ func TestValidateListArtifactsRequest_Error(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateGetArtifactRequest_Ok(t *testing.T) {
+	testData := []struct {
+		name    string
+		request request.GetArtifactRequest
+	}{
+		{
+			name: "NotEmptyPathCase1",
+			request: request.GetArtifactRequest{
```

Comment:
can you please add another case with `RunUUID`.

Reply:
added RunUUIDAndNotEmptyPath